themes/10up-theme/assets/js/frontend/components/navigation.js

```diff
@@ -0,0 +1,5 @@
+export default class Navigation {
```

Comment:
Does this actually need to be a class?

Reply:
It does not, but I think the large majority of our projects are still using class based components. I added a [comment](https://github.com/10up/wp-scaffold/pull/86/files#diff-d35de96f686e5dbc10d19b9db2b20419ac339e0e5d2d04c8d3ea5618ecc6de90R23-R26) for sites with functional components.

Reply:
@nicholasio this was just added for example sake - as long as this component is the default export we should be okay. Happy to update it to a `const Navigation = () => {}`